spec/aws/elb/idle_timeout_seconds.go

```diff
@@ -0,0 +1,11 @@
+package elb
+
+// AWS contains the Idle Timeout in seconds of the elastic load balancers.
+type AWS struct {
+	// API is the Idle Timeout in seconds for the Kubernetes API Server load balancer.
+	API int `json:"api" yaml:"api"`
+	// Etcd is the Idle Timeout in seconds for the etcd load balancer.
+	Etcd int `json:"etcd" yaml:"etcd"`
+	// Ingress is the Idle Timeout in seconds for the Ingress load balancer, used for customer traffic.
+	Ingress int `json:"ingress" yaml:"ingress"`
+}
```

Comment:
Here it should be.

```go
type ELB struct {
	API     elb.ELB `json:"api" yaml:"api"`
	Etcd    elb.ELB `json:"etcd" yaml:"etcd"`
	Ingress elb.ELB `json:"ingress" yaml:"ingress"`
}
```

Reply:
Hi Ross,  the yaml structure is like this:
```
  aws:
    region: "{{.Values.aws.region}}"
    az: "{{.Values.aws.az}}"
    elb:
      idleTimeoutSeconds:
        api: 120
        etcd: 60
        ingress: 60
```
So these field should be `int` unless you prefer a different yaml structure.

The name of the IdleTimeoutSeconds struct has a typo that I will change (AWS -> IdleTimeoutSeconds)

I'm conflicted with either the above structure, or using:
```
  aws:
    region: "{{.Values.aws.region}}"
    az: "{{.Values.aws.az}}"
    elb:
      api:
        idleTimeoutSeconds: 120
      etcd:
        idleTimeoutSeconds: 120
      ingress:
        idleTimeoutSeconds: 120
```
Do you have a preference?

Reply:
Ah yes I see what you mean! Let's go with the first format. It's less verbose but still easy to read.